oasis/access-control.md

```diff
@@ -421,12 +421,21 @@ Deployment Z of project DEF                  | ✓ | — | —
 
 ## Restricting access to organizations
 
-To enhance security, you can limit which authentication providers are accepted for users trying to access an organization in Oasis.
+To enhance security, you can implement the following restrictions via [Oasisctl](oasisctl.html):
 
-To do this, you need to use [Oasisctl](oasisctl.html). The following commands are available:
+1. Limit allowed authentication providers.
+2. Specify an allowed domain list.
 
-- `get organization authentication providers` - allows you to see which authentication providers are enabled for accessing a specific organization
-- `update organization authentication providers` - allows you to update a list of authentication providers for an organization to which the authenticated user has access
+Using the first option, you can limit which **authentication providers** are accepted for users trying to access an organization in Oasis. The following commands are available:
```

Comment:
...are available to configure this option:

Reply:
Done.